DataDefinitions/Modifications.cs

```diff
@@ -0,0 +1,89 @@
+            var PowerPlantArmoured = new Modifications("PowerPlant_Armoured");
+            var PowerPlantBoosted = new Modifications("PowerPlant_Boosted");
+            var PowerPlantStealth = new Modifications("PowerPlant_Stealth");
+            var SensorExpanded = new Modifications("Sensor_Expanded");
+            var SensorFastScan = new Modifications("Sensor_FastScan");
+            var SensorLightWeight = new Modifications("Sensor_LightWeight");
+            var SensorLongRange = new Modifications("Sensor_LongRange");
+            var SensorWideAngle = new Modifications("Sensor_WideAngle");
+            var ShieldBoosterExplosive = new Modifications("ShieldBooster_Explosive");
+            var ShieldBoosterHeavyDuty = new Modifications("ShieldBooster_HeavyDuty");
+            var ShieldBoosterKinetic = new Modifications("ShieldBooster_Kinetic");
+            var ShieldBoosterResistive = new Modifications("ShieldBooster_Resistive");
+            var ShieldBoosterThermic = new Modifications("ShieldBooster_Thermic");
+            var ShieldCellBankRapid = new Modifications("ShieldCellBank_Rapid");
+            var ShieldCellBankSpecialised = new Modifications("ShieldCellBank_Specialised");
+            var ShieldGeneratorKinetic = new Modifications("ShieldGenerator_Kinetic");
+            var ShieldGeneratorOptimised = new Modifications("ShieldGenerator_Optimised");
+            var ShieldGeneratorReinforced = new Modifications("ShieldGenerator_Reinforced");
+            var ShieldGeneratorThermic = new Modifications("ShieldGenerator_Thermic");
+            var WakeScannerFast = new Modifications("WakeScanner_Fast");
+            var WakeScannerLightWeight = new Modifications("WakeScanner_LightWeight");
+            var WakeScannerLongRange = new Modifications("WakeScanner_LongRange");
+            var WakeScannerReinforced = new Modifications("WakeScanner_Reinforced");
+            var WakeScannerShielded = new Modifications("WakeScanner_Shielded");
+            var WakeScannerWideAngle = new Modifications("WakeScanner_WideAngle");
+            var WeaponDoubleShot = new Modifications("Weapon_DoubleShot");
+            var WeaponEfficient = new Modifications("Weapon_Efficient");
+            var WeaponFocused = new Modifications("Weapon_Focused");
+            var WeaponHighCapacity = new Modifications("Weapon_HighCapacity");
+            var WeaponLightWeight = new Modifications("Weapon_LightWeight");
+            var WeaponLongRange = new Modifications("Weapon_LongRange");
+            var WeaponOvercharged = new Modifications("Weapon_Overcharged");
+            var WeaponRapidFire = new Modifications("Weapon_RapidFire");
+            var WeaponShortRange = new Modifications("Weapon_ShortRange");
+            var WeaponSturdy = new Modifications("Weapon_Sturdy");
+        }
+
+        public static readonly Modifications None = new Modifications("None");
```

Comment:
Ah. the initialisation needs to be moved into the class static ctor. This is because the class static ctor is guaranteed to run before all over class static methods. We leverage this to ensure that `AllOfThem` is fully initialised by the class static ctor.